eoscostarica/eos-rate
# Fix/proxy

### GH Issue

resolve #586 

### Steps to test

1. make run
1. make fresh
1. Check database population
